UI/window-basic-settings-stream.cpp

```diff
@@ -182,11 +182,14 @@ void OBSBasicSettings::LoadStream1Settings()
 
 #define SRT_PROTOCOL "srt"
 #define RIST_PROTOCOL "rist"
+#define WHIP_PROTOCOL "WHIP"
```

Comment:
```suggestion
#define WHIP_PROTOCOL "whip"
```

Unless this needs to be capitalized for other reasons.